OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs

```diff
@@ -22,20 +27,59 @@ public bool ValidateArguments(string[] args)
 			return args.Length >= 2;
 		}
 
-		[Desc("FILENAME", "Convert a legacy INI/MPR map to the OpenRA format.")]
+		[Desc("FILENAME", "Convert a legacy map to the OpenRA format guessing the correct mod.")]
 		public void Run(ModData modData, string[] args)
 		{
 			// HACK: The engine code assumes that Game.modData is set.
 			Game.ModData = modData;
 			Game.ModData.MountFiles();
 
-			var rules = Game.ModData.RulesetCache.Load();
-			var map = LegacyMapImporter.Import(args[1], modData.Manifest.Mod.Id, rules, Console.WriteLine);
+			var filename = args[1];
+			if (filename.ToLowerInvariant().EndsWith(".map"))
```

Comment:
The problem with hardcoded autodetection is that this then locks out any other map importers, because you need to edit this upstream file to make them work.  Can we please just stick with `--import-<mod>-map`?